src/aliases/api/handlers/handlers.go

```diff
@@ -0,0 +1,299 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+	"regexp"
+	"sync"
+
+	"github.com/consensys/quorum-key-manager/pkg/errors"
+	jsonutils "github.com/consensys/quorum-key-manager/pkg/json"
+	"github.com/consensys/quorum-key-manager/src/aliases/api/types"
+	aliasent "github.com/consensys/quorum-key-manager/src/aliases/entities"
+	infrahttp "github.com/consensys/quorum-key-manager/src/infra/http"
+	"github.com/gorilla/mux"
+)
+
+type AliasHandler struct {
+	alias aliasent.AliasBackend
+}
+
+func NewAliasHandler(backend aliasent.AliasBackend) *AliasHandler {
+	h := AliasHandler{
+		alias: backend,
+	}
+
+	return &h
+}
+
+func (h *AliasHandler) Register(r *mux.Router) {
+	regRoute := r.PathPrefix("/registries/{registry_name}").Subrouter()
+	regRoute.HandleFunc("", h.deleteRegistry).Methods(http.MethodDelete)
```

Comment:
Isn't it possible to list all available registries?

Reply:
Maybe in another PR
cc @nmvalera

Reply:
GTM